### ISSUE TITLE
android.content.res.Resources$NotFoundException: File res/drawable/exomedia_ic_skip_previous_white.xml from drawable resource ID #0x7f0200e8

### ISSUE THREAD

**Issue by pedro-almeida-clansoft** [brianwernick/ExoMedia#464]
- [X ] I have verified there are no duplicate active or recent bugs, questions, or requests

Maybe related to #270 

###### Include the following:
 - ExoMedia version: `4.0.2`
 - Device OS version: `4.4`
 - Devide Manufacturer: `ldpi Emulator (4.4) or LG`
 - Device Name: `L20 D100 - Android 4.4.2`
 
###### Reproduction Steps
 0.  Trying to instantiate a ExoMedia player with Custom controls results in crash

###### Expected Result
No crash

###### Actual Result
Crash

Stack Trace:
```
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.movile.meditation.vivo, PID: 5863
    java.lang.RuntimeException: Unable to start activity ComponentInfo{com.movile.meditation.vivo/com.movile.meditation.vivo.presentation.journey.view.JourneyActivity}: android.view.InflateException: Binary XML file line #109: Error inflating class ImageButton
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2195)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2245)
        at android.app.ActivityThread.access$800(ActivityThread.java:135)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1196)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:136)
        at android.app.ActivityThread.main(ActivityThread.java:5017)
        at java.lang.reflect.Method.invokeNative(Native Method)
        at java.lang.reflect.Method.invoke(Method.java:515)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:779)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:595)
        at dalvik.system.NativeStart.main(Native Method)
     Caused by: android.view.InflateException: Binary XML file line #109: Error inflating class ImageButton
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:714)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:756)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:759)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:759)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:492)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:397)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:353)
        at android.view.View.inflate(View.java:17465)
        at com.devbrackets.android.exomedia.ui.widget.VideoControls.setup(VideoControls.java:655)
        at com.devbrackets.android.exomedia.ui.widget.VideoControls.<init>(VideoControls.java:162)
        at com.movile.meditation.vivo.presentation.journey.video.view.customcontrols.CustomVideoControlsMobile.<init>(CustomVideoControlsMobile.kt:43)
        at com.movile.meditation.vivo.presentation.journey.video.view.customcontrols.CustomVideoControlsMobile.<init>(CustomVideoControlsMobile.kt:43)
        at com.movile.meditation.vivo.presentation.journey.video.view.JourneyVideoPlayerFragment.onActivityCreated(JourneyVideoPlayerFragment.kt:95)
        at android.support.v4.app.Fragment.performActivityCreated(Fragment.java:2201)
        at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1326)
        at android.support.v4.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1528)
        at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1595)
        at android.support.v4.app.BackStackRecord.executeOps(BackStackRecord.java:758)
        at android.support.v4.app.FragmentManagerImpl.executeOps(FragmentManager.java:2363)
        at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2149)
        at android.support.v4.app.FragmentManagerImpl.optimizeAndExecuteOps(FragmentManager.java:2103)
        at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2013)
        at android.support.v4.app.FragmentController.execPendingActions(FragmentController.java:388)
        at android.support.v4.app.FragmentActivity.onStart(FragmentActivity.java:607)
        at android.support.v7.app.AppCompatActivity.onStart(AppCompatActivity.java:178)
        at com.movile.standards.app.support.MAppCompatActivity.onStart(MAppCompatActivity.java:20)
        at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1171)
        at android.app.Activity.performStart(Activity.java:5241)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2168)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2245) 
        at android.app.ActivityThread.access$800(ActivityThread.java:135) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1196) 
        at android.os.Handler.dispatchMessage(Handler.java:102) 
        at android.os.Looper.loop(Looper.java:136) 
        at android.app.ActivityThread.main(ActivityThread.java:5017) 
        at java.lang.reflect.Method.invokeNative(Native Method) 
        at java.lang.reflect.Method.invoke(Method.java:515) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:779) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:595) 
        at dalvik.system.NativeStart.main(Native Method) 
     Caused by: android.content.res.Resources$NotFoundException: File res/drawable/exomedia_ic_skip_previous_white.xml from drawable resource ID #0x7f0200e8
        at android.content.res.Resources.loadDrawable(Resources.java:2101)
        at android.content.res.TypedArray.getDrawable(TypedArray.java:602)
        at android.widget.ImageView.<init>(ImageView.java:129)
    	at android.wi
```

**Comment by brianwernick** [comment]
I've verified again that it works on KitKat (and earlier) devices.

**Comment by pedro-almeida-clansoft** [comment]
Hi Brian,
thanks for quick reply. I'll further investigate this issue them. Thanks!

**Comment by pedro-almeida-clansoft** [comment]
Brian,
I've found the problem. In the project I was working there was a custom video control implementation. The layout file was using some bundled resources with the default exomedia library. Fortunately these resources weren't needed at our project, so I just had to remove the reference to them. Still, I don't know why this issue was happening only in devices with API <= 19.

Thanks again for your quickly reply!

Best,
